common/src/main/java/com/wynntils/utils/StringUtils.java

```diff
@@ -130,4 +130,8 @@ public static String toSignedString(int value) {
             return Integer.toString(value);
         }
     }
+
+    public static String stripMarkdownStyling(String input) {
+        return input.replaceFirst("[# ]*", "");
```

Comment:
Can't we replace this with `§` color codes? It looks very bland, and actually a bit hard to read. Would be nice if headings and bullet items had different colors.

Reply:
Fixed